corehq/apps/data_interfaces/models.py

```diff
@@ -1161,7 +1161,7 @@ def __init__(self, case, scheduler_module_info):
         self.scheduler_module_info = scheduler_module_info
 
     @classmethod
-    @quickcache(['domain', 'app_id', 'form_unique_id'], timeout=60 * 60)
+    @quickcache(['domain', 'app_id', 'form_unique_id'], timeout=60 * 60, memoize_timeout=60, session_function=None)
```

Comment:
Does [Giovanni's admonition](https://github.com/dimagi/commcare-hq/pull/19382) apply?

> You really should only use `memoize_timeout` in special circumstances when you understand the implications.

I'm guessing it does not matter here since workers never clear the cache (after mutating things that affect the result). I'm also assuming that stale results (inconsistencies between worker processes) are unlikely and probably not a big deal if they do occur.

Reply:
👍 It sounds like this is a reasonable use of `memoize_timeout` if you think stale cache data is not likely to cause issues here.

> For most background / non-user-facing processes having a few seconds of delay in cache refresh is not really an issue.

This is probably true most of the time, but caching is hard. https://github.com/dimagi/commcare-hq/pull/24987 comes to mind. Things get more scary when new data is written based on stale cache results.

Reply:
Not that I'm not changing the fact that there will be stale cache data here. The only difference is that it may be stale for an extra 1 minute (out of 60).

I also do not think that stale data is a problem here provided it doesn't remain stale for and extended period of time (like hours or days)